apps/meteor/app/livechat/server/api/v1/contact.ts

```diff
@@ -82,3 +86,15 @@ API.v1.addRoute(
 		},
 	},
 );
+
+API.v1.addRoute(
+	'omnichannel/contacts',
+	{ authRequired: true, permissionsRequired: ['view-l-room'], validateParams: isPOSTOmnichannelContactsProps },
+	{
+		async post() {
+			const contactId = await createContact({ ...this.bodyParams, unknown: false });
```

Comment:
Curious why `unknown` is always false here? 🤔

Reply:
When a contact is manually created, it will always be known. But in the future, when a new visitor automatically creates a contact (by openning a conversation), it will be an unknown contact.